src/commands.ts

```diff
@@ -51,3 +51,42 @@ export const resetCommand = async () => {
 	info("Biome extension was reset");
 	await start();
 };
+
+export const initializeWorkspaceCommand = async () => {
+	const subconfigs = [
+		"[javascript]",
+		"[typescript]",
+		"[typescriptreact]",
+		"[javascriptreact]",
+		"[json]",
+		"[jsonc]",
```

Comment:
We should add `vue`, `astro`, `svelte`, `css` and `graphql`

Reply:
added it in as well

Reply:
@ematipico If this means Biome will take precedence over the built-in extensions for vue/astro/svelte files, I'm not sure that's a great idea until support has progressed further and stabilized. I suspect anyone using these frameworks in earnest will prefer the official extensions at the moment, which means we'll need to document how to remove these lines from `settings.json`/.`code-workspace` (which is the kind of complexity that tends to intimidate new users).

Reply:
that's a valid point, I left it generic in the beginning as I wasn't sure if I should add specific languages as well but yeah this is definitely something to think about from the maintainers side of view!

Reply:
I think @colinhacks makes a good point here. We should only include languages for which we have full formatting support.

This would mean removing `vue`, `svelte`, and `astro` for now.